numba_cuda/numba/cuda/tests/cudapy/test_debuginfo.py

```diff
@@ -403,6 +403,387 @@ def f(x, y):
         match = re.compile(pat6).search(llvm_ir)
         self.assertIsNotNone(match, msg=llvm_ir)
 
+    def test_DW_LANG(self):
+        @cuda.jit(debug=True)
```

Comment:
No need to change things here (unless you want to, possibly in a follow-up), but if you specify a signature then the kernel will be compiled when it is defined for the given signature, which saves launching it as part of the test. For this kernel it would be

```suggestion
        @cuda.jit("void()", debug=True)
```

Then `foo[1, 1]()` below can be omitted and the LLVM IR will still be present for FileCheck to check.

Reply:
Ah, I see now that this pattern is used in `test_DITypes` below.